lib/shared/drive-constraints.js

```diff
@@ -405,6 +437,11 @@ exports.getDriveImageCompatibilityStatuses = (drive, image) => {
       type: exports.COMPATIBILITY_STATUS_TYPES.ERROR,
       message: exports.COMPATIBILITY_STATUS_MESSAGES.LOCKED
     })
+  } else if (exports.isDriveSizeLarge(drive)) {
```

Comment:
Should the unit-tests that deal with `getDriveImageCompatibilityStatuses` be updated too?

Reply:
Done